data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -222,6 +285,8 @@ SELECT
   )) AS publisher_json,
   license.license_url AS license,
   license.license_timestamp,
-FROM t_result_with_preprint_published_at_date_and_tdm_path AS result
-LEFT JOIN t_latest_manuscript_license AS license
-ON result.long_manuscript_identifier = license.long_manuscript_identifier
+FROM t_result_with_preprint_version AS result
+  LEFT JOIN t_result_with_preprint_details_array AS preprint_detail
```

Comment:
Usually the `LEFT JOIN` isn't indented, but the `ON` is. There are a few inconsistent formatting in the file.